desktop/dev-environments.md

```diff
@@ -253,6 +253,21 @@ In this preview, Dev Environments support a JSON file which allows you to specif
 }
 ```
 
+Next, you have to define the dependencies you want to include in your `Dockerfile.devenv`, alongside the following requisites:
+
+While any images or Dockerfiles will include a non-root user with a UID/GID of `1000`, many base images and Dockerfiles do not. Fortunately, you can add a non-root user named `vscode`. If you were to include the Docker tooling (e.g. `docker` cli, `docker compose`, etc.) in the `Dockerfile.devenv`, you would need the `vscode` user to be included in the `docker` group. Finally, remember to set `sleep infinity` in the entrypoint to keep the dev container running indefinitely.
```

Comment:
Should the example explicitly use `1000` as `uid` ?

Also wondering; is the `vscode` user required, or would numeric `uid/gid` work?;

```dockerfile
USER 1000:1000
```

Reply:
After a discussion with @glours, we want to avoid confusing the reader with the ID/GID `1000`. 

We'll remove that part and just mention adding a non-root user (regardless of the ID/GID) named `vscode`.